docs/guides/migrating-to-v2.mdx

```diff
@@ -26,6 +26,24 @@ Other notable changes to keep things updated and fresh include:
 - Requirement of TypeScript version 4.5 and above
 - The build target updated to ES2018
 
+## Codemods
```

Comment:
Thanks for your suggestion! However, I would like to move this section after `## Migration` and remove the change in the migration section. Also, we may need to add some caveats.

Here're some thoughts behind it:
- I don't think most users need migration.
- Only a few users might depend on the subtle behavioral difference, but they should reconsider their code.
- Doing codemods blindly may result in bad performance.
- I haven't looked into the codemode impl, but it may not work for some cases, because we can't statically know what are promises.